docs/usage/tables/avro.md

```diff
@@ -0,0 +1,105 @@
+
+After you run the `table create` command, your output should look something like this:
+
+```
+┌────────────────────┬──────────────────────────────────────────────────────────────────────────────────────────┐
+│        KEY         │                                          VALUE                                           │
+├────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────┤
+│NAME                │test                                                                                      │
+├────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────┤
+│CATALOG_NAME        │unity                                                                                     │
+├────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────┤
+│SCHEMA_NAME         │default                                                                                   │
+├────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────┤
+│TABLE_TYPE          │EXTERNAL                                                                                  │
+├────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────┤
+│DATA_SOURCE_FORMAT  │AVRO                                                                                      │
+├────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────┤
+│COLUMNS             │{"name":"some_numbers","type_text":"int","type_json":"{\"name\":\"some_numbers\",\"type\":│
+│                    │\"integer\",\"nullable\":true,\"metadata\":{}}","type_name":"INT","type_precision":0,"type│
+│                    │_scale":0,"type_interval_type":null,"position":0,"comment":null,"nullable":true,"partition│
+│                    │_index":null}                                                                             │
+│                    │{"name":"some_letters","type_text":"string","type_json":"{\"name\":\"some_letters\",\"type│
+│                    │\":\"string\",\"nullable\":true,\"metadata\":{}}","type_name":"STRING","type_precision":0,│
+│                    │"type_scale":0,"type_interval_type":null,"position":1,"comment":null,"nullable":true,"part│
+│                    │ition_index":null}                                                                        │
+├────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────┤
+│STORAGE_LOCATION    │file:///Users/avriiil/Documents/git/my-forks/unitycatalog/etc/data/external/unity/default│
```

Comment:
Fix the output to match the command.

Reply:
This is the output copied from my terminal after running the command. I have pasted it again just to be sure.
Can you perhaps specify what seems incongruous?